TestAssets/CrossPublishTestProjects/StandaloneAppCrossPublish/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
+      "dependencies": {
```

Comment:
I copied the standalone app present in PortableTests. 

I agree we could use cleanup all around in the TestAssets but i think it's out of the scope of this PR.